policyengine_us/parameters/gov/states/mt/tax/income/deductions/standard/max.yaml

```diff
@@ -0,0 +1,26 @@
+description: Montana provides filers a maximum standard deduction of this amount, depending on filing status.
+metadata:
+  label: Montana standard deduction max amount
+  period: year
+  unit: currency-USD
+  breakdown:
+    - filing_status
+  reference:
+    - title: MT Code, Income Tax - Deductions, § 42-15-512 (2)
```

Comment:
The values here are not in this reference. Please add tax form references to ensure all values have a reference. Same for min.